media/stitcher/api/Stitcher.Samples/UpdateSlate.cs

```diff
@@ -0,0 +1,51 @@
+﻿/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// [START video_stitcher_update_slate]
+
+using Google.Api.Gax.ResourceNames;
+using Google.Cloud.Video.Stitcher.V1;
+using Google.Protobuf.WellKnownTypes;
+
+public class UpdateSlateSample
+{
+    public Slate UpdateSlate(
+        string projectId, string location, string slateId, string slateUri)
+    {
+        // Create the client.
+        VideoStitcherServiceClient client = VideoStitcherServiceClient.Create();
+
+        // Build the parent location name.
+        LocationName parentLocation = new LocationName(projectId, location);

```

Comment:
You are not using this here. Check in other samples as well.
```suggestion

```

Reply:
Removed